src/interceptors/ClientRequest/MockHttpSocket.ts

```diff
@@ -63,6 +63,7 @@ export class MockHttpSocket extends MockSocket {
   private responseType: 'mock' | 'bypassed' = 'bypassed'
   private responseParser: HTTPParser<1>
   private responseStream?: Readable
+  private isPassthrough: boolean
```

Comment:
Cannot this be represented via `responseType === 'bypassed`? We already have a flag to track this.